tests/test_transform.py

```diff
@@ -180,10 +180,94 @@ def checkTransform(self, fromName, toName):
                 with self.assertRaises(InvalidParameterError, msg=msg):
                     transformClass(badPolyMap)
 
+    def checkGetInverse(self, fromName, toName):
+        """Test Transform<fromName>To<toName>.getInverse
+
+        Parameters
+        ----------
+        fromName, toName : string
+            the prefixes of the transform's endpoints (e.g., "Point2" for a
+            Point2Endpoint)
+        """
+        transformClassName = "Transform{}To{}".format(fromName, toName)
+        transformClass = getattr(afwGeom, transformClassName)
+        baseMsg = "transformClass={}".format(transformClass.__name__)
+        for nIn in self.goodNaxes[fromName]:
+            for nOut in self.goodNaxes[toName]:
+                # Transform with both forward and inverse transforms
+                self.checkGetInverseWithMap(
+                    transformClass(makePolyMap(nIn, nOut)),
+                    nIn,
+                    "{}, Map={}, nIn={}, nOut={}".format(
+                        baseMsg, "Invertible", nIn, nOut))
+                # TODO: add more test cases once DM-9826 resolved
```

Comment:
I was trying to be better about organizing work, avoiding scope creep, etc., but fine. I will ask for a re-review of those changes, though.